met-web/src/components/map/index.tsx

```diff
@@ -10,8 +10,8 @@ import { When } from 'react-if';
 import { AnyLayer } from 'mapbox-gl';
 
 interface MapProps {
-    latitude: number;
-    longitude: number;
+    latitude?: number;
+    longitude?: number;
```

Comment:
what happens when we have lat and long undefined and pass them like that to the map?